tickets/src/__tests__/components/developer/DeveloperOverlay.test.js

```diff
@@ -40,17 +40,17 @@ describe('DeveloperOverlay', () => {
   })
 
   it('has a dropdown that can be used to choose between providers', () => {
-    expect.assertions(2)
+    expect.assertions(1)
     const component = rtl.render(
       <DeveloperOverlay config={config} setProvider={callback} />
     )
 
-    expect(component.queryByText('HTTP')).not.toBeNull()
+    component.getByText('HTTP')
     expect(component.queryByText('Metamask')).not.toBeNull()
   })
 
   it('sets selected provider from prop', () => {
-    expect.assertions(1)
+    expect.assertions(0)
```

Comment:
That's a bit strange? I think we should have at leaast one expectation?

Reply:
`getBy<query name>('a string')` fails the test if it doesn't find the element, so it works about the same as `expect(queryBy<query name>('a string')).not.toBeNull()` but doesn't contain an `expect`.

Reply:
I understand but it's strange to have this `expect.assertions(0)`
Let's unlblock for now though.